components/hab/src/cli.rs

```diff
@@ -351,21 +350,49 @@ pub fn get() -> App<'static, 'static> {
         )
         (subcommand: alias_apply)
         (subcommand: alias_install)
+        (subcommand: alias_run())
         (subcommand: alias_setup)
-        (subcommand: alias_start)
+        (subcommand: alias_start())
+        (subcommand: alias_stop())
+        (subcommand: alias_term())
         (after_help: "\nALIASES:\
             \n    apply      Alias for: 'config apply'\
             \n    install    Alias for: 'pkg install'\
+            \n    run        Alias for: 'sup run'\
             \n    setup      Alias for: 'cli setup'\
             \n    start      Alias for: 'sup start'\
+            \n    term       Alias for: 'sup term'\
             \n"
         )
     )
 }
 
+fn alias_run() -> App<'static, 'static> {
+    clap_app!(@subcommand run =>
+        (about: "Run the Habitat Supervisor")
+        (@setting Hidden)
+    )
+}
+
 fn alias_start() -> App<'static, 'static> {
     clap_app!(@subcommand start =>
         (about: "Starts a Habitat-supervised service")
+        (aliases: &["st", "sta", "star"])
```

Comment:
Might need to drop the `"st"` (I see it was there before)--it would conflict with `hab studio`. One of these days I'll see about adding this functionality upstream